docs/advanced/document-store/index.md

```diff
@@ -201,3 +201,65 @@ da[0, 'text'] = 'hello'
 Obviously, a DocumentArray with on-disk storage is slower than in-memory DocumentArray. However, if you choose to use on-disk storage, then often your concern of persistence overwhelms the concern of efficiency.
  
 Slowness can affect all functions of DocumentArray. On the bright side, they may not be that severe as you would expect. Modern database are highly optimized. Moreover, some database provides faster method for resolving certain queries, e.g. nearest-neighbour queries. We are actively and continuously improving DocArray to better leverage those features. 
+
+
+## Benchmarking
```

Comment:
please move to a dedicated page, under "Add New Document Store"

<img width="284" alt="image" src="https://user-images.githubusercontent.com/2041322/163364399-2f786b8c-26ea-4eab-8703-7b298a35af33.png">

Reply:
done